controllers/operator/openstack_controller.go

```diff
@@ -367,6 +382,65 @@ func (r *OpenStackReconciler) renderAndApply(
 	return nil
 }
 
+var serviceOperatorNames = []string{"barbican", "cinder", "designate", "glance", "heat", "horizon", "infra",
```

Comment:
This probably doesn't need to be heap allocated. We're only using it in `isServiceOperatorResource`. I know that function looks nice and tidy as it is, but from a memory management perspective, it would be more efficient and clean if this were stack allocated where it's being referenced.

Reply:
I think it would be worth adding a comment here that this list static and will not require updates in the future due to there not being backported operators. The only other thing I can think of though is that it _might_ update to add watcher perhaps once that is rolled in?

Reply:
@bshephar will move inside the function
@leifmadsen will add a comment according to your suggestion
thanks for the feedback